releasing.md

```diff
@@ -0,0 +1,304 @@
+        type commit
+        tag 0.2.1
+        tagger Matthias Wessendorf <matzew@apache.org> 1375864277 +0200
+
+        signed unified push java client 0.2.1 tag
+        gpg: Signature made Wed Aug  7 10:31:17 2013 CEST using DSA key ID 1CE17EDC
+        gpg: Good signature from "Matthias Wessendorf <matzew@apache.org>"
+```
+
+   * (Optional) Go to your Github profile page and add or make sure you GPG public key is added. Going to [release tab](https://github.com/aerogear/aerogear-parent/releases/tag/0.2.21) you will show a `Verified` label.
+   * If all went OK, push the tag
+```
+git push origin x.y.z
+```
+where x.y.z is the release tag.
+
+**Note:** The delete and recreate tasks are a workaround of this [bug](http://jira.codehaus.org/browse/SCM-486) .
+
+**Note 2:** You don't need to push the changes on master, only the tag. (already tested)
+
+#### Perform the Release
+
+While you're still on master branch, do this:
+
+```
+        MAVEN_OPTS="-DperformRelease=true -Dgpg.useagent=true" mvn release:perform
+```
+
+#### Stage on Nexus
+
+Go to the staging repository and check if the artifact was corrected uploaded.
+
+* Go to https://oss.sonatype.org/#stagingRepositories
+* There must be a repository called something like `orgjbossaerogear-1001` which is implicitly created by auto staging
+* Check the summary, activity and content and make sure things are correct
+* In the content, make sure you can see sources and javadocs are published as well
+* In the content, make sure you can see `.asc` files for every published artifact
+
+These are some screenshots:
+![nexus screenshot 1](release_screenshot01.png)
+![nexus screenshot 2](release_screenshot02.png)
+![nexus screenshot 3](release_screenshot03.png)
+
+
+* Close the repository to stage ("close" could sound confusing but in fact it means your repository will be staged).
+![nexus screenshot 4](release_screenshot04.png)
+
+
+* Press the "Refresh" button next to "Close" to check the status. Eventually, if everything is fine,
+  you will see this:
+![nexus screenshot 5](release_screenshot05.png)
+
+
+* Communicate on the mailing list the repository URL.
+
+* Be sure to have git-extras installed, this way you can easily generate a change log by running ```git changelog``` , that will create an history.md file containing your changelog.
+
+In the body of your email make sure to mention :
+* The new version that has been released
+* The staging repo location (sth like https://oss.sonatype.org/content/repositories/orgjbossaerogear-1001, shown above in the screenshots)
+* The changelog
+
+
+Now, wait until the staging is done, tests have been successful and community gave approval (or gave no feedback within 72 hours).
+
+
+#### Push the changes performed by maven plugin
+
+**Note:** You only do this step once the staging is done, tests have been successful and community gave approval (or gave no feedback within 72 hours).
+
+```
+git push origin master
```

Comment:
for that you need to disable the auto push setting

Reply:
Well, in my tests, nothing were pushed automatically. So, I assume it is disabled on either AeroGear parent or JBoss parent